scripts/known_good.json

```diff
@@ -6,15 +6,15 @@
             "sub_dir": "Vulkan-Headers",
             "build_dir": "Vulkan-Headers/build",
             "install_dir": "Vulkan-Headers/build/install",
-            "commit": "v1.3.257"
+            "commit": "v1.3.259"
```

Comment:
Is this exepected?

Reply:
Forgot to include this file with the latest push, fixed now